src/quicklook.cs

```diff
@@ -119,6 +128,19 @@ interface QLPreviewControllerDelegate {
 		[Export ("previewController:transitionViewForPreviewItem:"), DelegateName ("QLTransitionView"), DefaultValue (null)]
 		[return: NullAllowed]
 		UIView TransitionViewForPreviewItem (QLPreviewController controller, IQLPreviewItem item);
+
+		[iOS (13,0)]
+		[Export ("previewController:editingModeForPreviewItem:"), DelegateName("QLEditingMode"), DefaultValue ("QLPreviewItemEditingMode.Disabled")]
+		QLPreviewItemEditingMode EditingModeForPreviewItem (QLPreviewController controller, IQLPreviewItem item);
```

Comment:
prefix: use a verb for methods, e.g. `Get...`
suffix: avoid repeating parameter names, e.g. drop `ForPreviewItem`

so `GetEditingMode`

note: ObjC selectors are `{method_name}{param1}:{param2}` - i.e. there's no `:` between the method name and the **first** parameter name. In doubt check the name used by Apple for the Swift API.

Reply:
This also means the parameter can be better named as `previewItem`:

```csharp
QLPreviewItemEditingMode GetEditingMode (..., IQLPreviewItem previewItem);
```

Reply:
Updated this and others! Thanks @spouliot and @rolfbjarne :)